IPCheck/parseThemAll.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+./parseAll.sh "/Users/philippe/git" "/Users/philippe/temp/smartclide/analysis"
```

Comment:
Maybe put `parseThemAll.sh` and `parseThisOne.sh` as examples into a `README.md` file?

I would suggest not to include them as source files - at least not as they are now - because they contain file paths that are specific for your machine.